apps/meteor/client/providers/TooltipProvider.tsx

```diff
@@ -46,22 +46,20 @@ const TooltipProvider: FC = ({ children }) => {
 				setTooltip(<TooltipComponent title={title} anchor={anchor} />);
 			}, 300);
 			setTooltip(null);
-		};
 
-		const handleClick = (): void => {
-			setTooltip(null);
-			clearTimeout(timeout);
+			target.onclick = (): void => {
+				setTooltip(null);
+				clearTimeout(timeout);
+			};
```

Comment:
nop, there reason to to this? I'm afraid this going to pollute the html and it doesnt work like before